### PR TITLE
PLATFORM-3369: send beacon errors to event-logger

### PR DESCRIPTION
**Pull request by themech**
We don't use loggly, so switching to event-logger service.
Also services urls are generated when needed based on a shared `$wgServicesExternalDomain` variable introduced in https://github.com/Wikia/config/pull/2403

### CODE REVIEW

File: extensions/wikia/Track/templates/track.mustache [hunk 1]
```diff
@@ -29,13 +29,17 @@
...
-			document.body.appendChild(image);
-		});
+		try {
+			var request = new XMLHttpRequest(),
+				data = {
+					name: 'MediaWiki.track',
+					description: 'wikia_beacon_id is empty',
+					client: 'oasis'
+				};
+			request.open('POST', '{{{event-logger-url}}}/error', true);
```
Comment (Grunny) [inline extensions/wikia/Track/templates/track.mustache]: We should default to HTML escaping on output, even if it's a hard coded string, so `{{event-logger-url}}` would be preferred.

File: extensions/wikia/Discussions/Discussions.setup.php [hunk 2]
```diff
@@ -66,3 +66,5 @@
 $wgGroupPermissions['staff']['specialdiscussions'] = true;
 
 $wgHooks['WikiaSkinTopScripts'][] = 'DiscussionsHooksHelper::addDiscussionJsVariable';
+
+$wgDiscussionsApiUrl = $wgServicesExternalDomain . 'discussion';
```
Comment (Grunny) [inline extensions/wikia/Discussions/Discussions.setup.php]: Should this one use the URL provider wherever it is used as well?

**Review by Grunny — CHANGES_REQUESTED** [review]

File: extensions/wikia/Discussions/Discussions.setup.php (hunk 2 above)
Comment (harnash) [reply extensions/wikia/Discussions/Discussions.setup.php]: fix: 3c073840caa

File: extensions/wikia/Track/templates/track.mustache (hunk 1 above)
Comment (harnash) [reply extensions/wikia/Track/templates/track.mustache]: fix: 7f0c497dfd4

**Review by Grunny — APPROVED** [review]

**Pull request merged**